pepper-apis/src/main/java/org/broadinstitute/ddp/model/governance/GovernancePolicy.java

```diff
@@ -73,12 +73,12 @@ public void addAgeOfMajorityRule(AgeOfMajorityRule... rules) {
     }
 
     public boolean shouldCreateGovernedUser(Handle handle, PexInterpreter interpreter, String userGuid) {
-        return interpreter.eval(shouldCreateGovernedUserExpr.getText(), handle, userGuid, null);
+        return interpreter.eval(shouldCreateGovernedUserExpr.getText(), handle, userGuid, userGuid, null);
```

Comment:
It might be more correct to pass in the operator guid here.

Reply:
Governance policy is being checked during operator registration. So userGuid is actually always operatorGuid here. You can check the calls of this method.